src/Particular.CodeRules/tools/install.ps1

```diff
@@ -0,0 +1,6 @@
+﻿param($installPath, $toolsPath, $package, $project)
```

Comment:
The install script needs to run to be able to add an analyzer reference - it's not a normal project reference. To be honest I haven't dug too deep to see if it would be possible to install analyzers without this. Perhaps @distantcam would have an idea.

I'm not sure what the harm of checking this into source control is though?